scripts/zkai_attention_kv_d16_native_width_gate.py

```
@@ -0,0 +1,626 @@
+        "score_rows_held_constant": 52,
+        "trace_rows_held_constant": 64,
+        "selected_positions_changed": True,
+    }
+    if payload["width_axis_result"] != expected_axis:
+        raise AttentionKvD16NativeWidthGateError("width-axis result drift")
+    if payload["scale_gate_commitment"] != _expected_commitment(payload):
+        raise AttentionKvD16NativeWidthGateError("scale gate commitment drift")
+    if not allow_missing_mutation_summary:
+        cases = payload["mutation_cases"]
+        if not isinstance(cases, list) or [case.get("name") for case in cases] != list(EXPECTED_MUTATION_NAMES):
+            raise AttentionKvD16NativeWidthGateError("mutation case drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)